Dockerfile

```
@@ -1,15 +1,39 @@
+ARG HELM_DOCS_VERSION=1.14.2
+ARG POETRY_VERSION=2.2.1
 
-RUN apt update && apt install curl git -y
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends curl git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
-COPY . .
+# Install helm-docs with checksum verification
+RUN cd /tmp && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)